c_glib/arrow-glib/array-builder.cpp

```diff
@@ -4182,6 +4182,32 @@ gint64 garrow_string_dictionary_array_builder_get_dictionary_length(GArrowString
   return arrow_builder->dictionary_length();
 }
 
+/**
+ * garrow_string_dictionary_array_builder_finish_delta:
+ * @builder: A #GArrowStringDictionaryArrayBuilder.
+ * @out_delta: (transfer full): The built #GArrowArray containing dictionary.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: (transfer full): The built #GArrowArray containing indices on
+ *   success, %NULL on error.
+ *
+ * Since: 1.0
+ */
+GArrowArray *
+garrow_string_dictionary_array_builder_finish_delta(GArrowStringDictionaryArrayBuilder* builder,
+                                                    GArrowArray **out_delta,
```

Comment:
@kou I don't know how to handle this `out_delta` in the test code written in Ruby.  Could you tell me the way to handle it?

Reply:
See d3395c9.

Reply:
Thanks!